gp/primitives.py

```
@@ -0,0 +1,27 @@
+import operator
+import random
+from deap import gp
+from . import utils
```

Pylint (3.3.7)

[error] No name 'utils' in module ''

(E0611)